rotkehlchen/chain/ethereum/modules/nft/nfts.py

```diff
@@ -209,7 +209,7 @@ def query_balances(
         with self.db.conn.read_ctx() as cursor:
             accounts = self.db.get_blockchain_accounts(cursor=cursor)
         # Be sure that the only addresses queried already exist in the database. Fix for #4456
-        queried_addresses = list(set(accounts.eth) & set(addresses))
+        queried_addresses = sorted(set(accounts.eth) & set(addresses))  # Sorting for consistency in tests  # noqa: E501
```

Comment:
We never ever change production to accomodate tests. **NEVER**.

But it's needed to have reproducible queries from API endpoints. So this is why you do this.

```suggestion
        queried_addresses = sorted(set(accounts.eth) & set(addresses))  # Sorting to remove randomness
```